packages/@ember/-internals/glimmer/lib/syntax/outlet.ts

```diff
@@ -89,6 +89,9 @@ export const outletHelper = internalHelper(
             return model;
           });
 
+          let componentRef = childRefFromParts(outletRef, ['render', 'component']);
+          named['component'] = componentRef;
+
           if (DEBUG) {
             named['model'] = createDebugAliasRef!('@model', named['model']);
           }
```

Comment:
Also not really sure what's up with this, does this just predate `createComputeRef` taking an optional label argument? (In any case the positioning of the new code split these two related chunks)